code/workspaces/web-app/src/components/assetRepo/AssetAccordion.js

```diff
@@ -84,11 +88,22 @@ export const onEditAssetConfirm = async (dispatch, asset) => {
   }
 };
 
+const allowEdit = (userId, userPermissions, asset) => {
+  // For now, only Data Managers can edit an asset.
+  if (userPermissions.value && userPermissions.value.includes(SYSTEM_DATA_MANAGER)) {
```

Comment:
could this just be `return userPermissions.value && userPermissions.value.includes(SYSTEM_DATA_MANAGER)` or is the return type of `includes` not a boolean?

Reply:
It could be that yeah, though as part of a follow on story (1008), this function will do some more checks involving the userId and asset as well.